plugins/task-plugin/src/task/che-task-runner.ts

```diff
@@ -59,13 +59,16 @@ export class CheTaskRunner {
             throw new Error("Che task config must have 'target.machineName' property specified");
         }
 
+        const workingDir = target.workingDir;
```

Comment:
Do we need this const here? As I can see we don't use it in another places, so you can just pass `target.workingDir`. But up to you.

Reply:
so many men, so many minds
Make sense   
```
const cwd = target.workingDir;
...
          tty: true,
          cwd
};
```